croud/cmd.py

```diff
@@ -203,7 +203,7 @@ def consumer_schema_arg(
     group.add_argument(
         "--consumer-schema",
         type=str,
-        help="The CrateDB database schema used by the Azure EventHub consumer.",
+        help="The CrateDB database schema used by the Azure Events endpoint consumer.",
```

Comment:
Shouldn't it be "Event Hubs" and not "Events" ?
https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about

```suggestion
        help="The CrateDB database schema used by the Azure Event Hubs endpoint consumer.",
```

Same for the other help texts as well?